Automattic/wp-calypso
# Launchpad: Add domain upgrade badge

#### Proposed Changes

* Adds a `Customize` badge next to the Launchpad sidebar URL for free domains.
If the sidebar URL is a paid domain (non WPCOM domain), then the 'Customize' badge does not appear.
The badge redirects the user to the `/my-site/` domains page.

#### Testing Instructions

* Check out this branch.
* Either create a site with a paid domain (https://wordpress.com/hp-2022-tailored-flows/) or use an existing Launchpad-enabled site with a paid domain available.
* Set the free domain (wpcom domain) to your primary address (https://wordpress.com/domains/manage/agrullon95.wordpress.com) and make sure the `Customize` badge appears next to the sidebar URL.
* Set the paid domain (non wpcom domain) to your primary address and make sure the `Customize` badge DOES NOT appear next to the sidebar URL.
* Ensure the launchpad test cases are passing: `yarn test-client client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test`

<img width="460" alt="image" src="https://user-images.githubusercontent.com/10482592/195900837-65a5f8ca-c4ae-46c6-927d-83ae60529077.png">




*

#### Pre-merge Checklist






Related to #68969
